Server/src/opcserver_open62541.cpp

```diff
@@ -74,7 +81,15 @@ int OpcServer::stop(OpcUa_Int32 secondsTillShutdown, const UaLocalizedText& shut
 
 void OpcServer::runThread()
 {
-    UA_Server_run(m_server, &g_RunningFlag);
+    while (g_RunningFlag)
+    {
+        UA_Server_run_iterate(m_server, true);
+    }
+    UA_StatusCode status = UA_Server_run_shutdown(m_server);
+    if (status != UA_STATUSCODE_GOOD)
+    {
+        LOG(Log::ERR) << "UA_Server_run_shuttdown returned not-good. Error was:" << UaStatus(status).toString().toUtf8();
+    }
```

Comment:
Might be nice to also log a message if the status is good. Shutting down the server sounds like a key moment in the server lifetime; sometimes it's nice to read a log message that (at this point anyway) the server was still on the happy path.

Reply:
OK, I don't mind, can be done.

Reply:
Pushed.